src/shared/components/plots/MultipleCategoryBarPlot.tsx

```diff
@@ -917,3 +929,47 @@ export default class MultipleCategoryBarPlot extends React.Component<
         );
     }
 }
+
+type QPLabelProps = {
+    x: number;
+    y: number;
+    pValue: number | null;
+    qValue: number | null;
+};
+
+const PQValueLabel = (props: QPLabelProps) => {
```

Comment:
I think it's better to give it a type like this, although it won't impact the performance, but it gives us a little bit more control on return type (e.g. this will force you to return a component instead of a string).
```
const PQValueLabel: React.FunctionComponent<PQLabelProps> = props => {
    // COMPONENT CONTENT
}

Reply:
Solved in [846942a](https://github.com/cBioPortal/cbioportal-frontend/pull/4234/commits/846942aa16d04fc787ad358d5004dfd3e43c7828)